k8s/charts/openebs/values.yaml

```diff
@@ -85,6 +85,8 @@ ndm:
     excludeVendors: "CLOUDBYT,OpenEBS"
     includePaths: ""
     excludePaths: "loop,fd0,sr0,/dev/ram,/dev/dm-,/dev/md"
+  probes:
```

Comment:
@akhilerm -- add this to the  README table also please.

Reply:
Added @kmova